test/base_results/upgrades/annual/results_output.csv

```diff
@@ -1,21 +1,23 @@
+project_national-ASHP.osw,1,0.0,7191.27,0.0,1522.25,0.0,8713.52,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.19,0.0,0.0,0.65,1.64,0.0,0.0,0.0,0.0,0.0,0.81,6.93,0.0,0.0,0.0,0.0,0.0,0.54,0.0,5.0,0.06,0.0,0.0,4.53,0.0,0.0,0.0,0.07,1.48,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,10.98,0.0,0.0,0.0,1.97,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,21.88,21.88,0.0,12.94,0.0,0.0,0.0,0.0,0.0,2345.58,6146.92,9.44,15.93,4.75,0.0,0.0,3.76,1527.75,6517.21,9.4,21.16,36.0,4.0,0.0,0.0,0.0,1138.0,1138.0,0.0,0.0,496.95,30.0,"HVAC Heating Efficiency|ASHP, SEER 22, 10 HSPF",,,,,,,,,,,,,,,,,,,,,,0.0,0.0,9.94,16.21,9.94,0.0,30.0,0.0,496.95,201.2,201.2,0.0,60.3,,,,18.71,17.01,19.0,1.12,1.78,1.06,0.32,0.48,0.24,19.27,14.73,1.38,5.37,810.0,6.52
```

Comment:
Why is `upgrade_costs.insulation_increase_floor_area_attic_delta_r_value_ft_2` zero for every row in this?

Reply:
Because the baseline sample has `Insulation Ceiling=None` (i.e., an apartment unit for which the upgrade doesn't apply).

Reply:
Would it be worth having say 5-10 baseline samples for evaluating upgrades?

Reply:
Dealing with that in a separate PR: https://github.com/NREL/resstock/pull/846.

Reply:
Cool. Maybe we can get that merged in first.

Reply:
OK, I think this is actually getting tested now using the example upgrade ymls.